src/types.ts

```diff
@@ -57,16 +57,16 @@ export interface sunsynkPowerFlowCardConfig extends LovelaceCardConfig {
         three_phase: boolean;
     }
     battery: {
-        energy: number;
-        shutdown_soc: number;
-        shutdown_soc_offgrid: number;
+        energy: any;
```

Comment:
Please confirm that I am corrct to change this to `any` where you can provide a number or sensor